src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/AbstractAppSchemaMockData.java

```diff
@@ -652,7 +660,7 @@ protected File getFeatureTypesBaseDir() {
      * @param typeName local name of the WFS feature type
      * @return directory that contains the mapping and property files
      */
-    private static File getFeatureTypeDir(
+    private File getFeatureTypeDir(
```

Comment:
Any specific reason to make this method no ``static`` anymore ?

Reply:
In this case, due to extended use along the abstract class for this method and the need to change its behavior on extended classes its needed to make the method non-static.